components/eam/cime_config/usermods_dirs/scm/shell_commands

```diff
@@ -0,0 +1,16 @@
+#!/bin/csh
+# need to use single thread
+foreach component ( ATM LND ICE OCN CPL GLC ROF WAV )
+ ./xmlchange  NTASKS_$component=1,NTHRDS_$component=1
```

Comment:
Can this also include `ROOTPE_$component=0` to force single-task runs on all machines (e.g. Anvil)?

Reply:
Is ROOTPE for each component set to non-zero on Anvil?

Reply:
Just checked the failed test on Anvil and the values don't all default to 0
```
> ./xmlquery ROOTPE
	ROOTPE: ['CPL:0', 'ATM:0', 'LND:108', 'ICE:0', 'OCN:108', 'ROF:108', 'GLC:0', 'WAV:0', 'IAC:0', 'ESP:0']
```

Reply:
I'll make a quick PR to resolve these two issues.

Reply:
great. thanks for checking.